calories/calories.rb

```diff
@@ -0,0 +1,84 @@
+require 'sqlite3'
+
+db = SQLite3::Database.new("daily_cal.db")
+db.results_as_hash = true
+
+continue = true
+$table_exists = false
+
+# allow user to add an item to the list.
+def add_item(db)
+	create_table_cmd = <<-SQL
+	CREATE TABLE IF NOT EXISTS items (
+		id INTEGER PRIMARY KEY,
+		name VARCHAR(255),
+		calories INT
+	)
+	SQL
+	db.execute(create_table_cmd)
+
+	$table_exists = true
+
+	puts "Please enter item name:"
+	item_name = gets.chomp
+	puts "Please enter calories for #{item_name}:"
+	cals = gets.chomp.to_i
+	db.execute("INSERT INTO items (name, calories) VALUES (? ,?)", [item_name, cals])
+	puts "Item added successfully!"
+end
+
+# allow user to see all the items in the list and the sum of calories consumed. 
+def summary(db)
+	if $table_exists
```

Comment:
What if you updated you global variable somewhere else by mistake?  This could cause a break that would be very difficult to find.